superset/viz.py

```diff
@@ -439,11 +439,7 @@ def get_df_payload(self, query_obj=None, **kwargs):
                 and self.status != utils.QueryStatus.FAILED
             ):
                 try:
-                    cache_value = dict(
-                        dttm=cached_dttm,
-                        df=df if df is not None else None,
```

Comment:
`df if df is not None else None` is merely `df`.

Reply:
Very true.

Reply:
![image](https://user-images.githubusercontent.com/33317356/72008984-a03e3480-325d-11ea-86ea-924887888bd1.png)